### PR TITLE
Fix rsync

### PR DESCRIPTION
**Pull request by Jokeswar**
vagrant.vm.synced_folder wasn't using the correct argument name for rsync.

### CODE REVIEW

**Comment by gabriel-v** [comment]
@Jokeswar please rebase this PR so tests pass

**Comment by Jokeswar** [comment]
I changed `.gitignore` because rsync seems to not agree with the way it was written. It simply ignored the whole `./bin` folder instead of keeping the `.gitignore` and `docker.sh`.

File: ci/Vagrantfile [hunk 1]
```diff
@@ -11,8 +11,8 @@ Vagrant.configure("2") do |config|
   config.vm.synced_folder '.', '/vagrant', disabled: true
   config.vm.synced_folder "..", "/opt/cluster", type: "rsync",
       rsync__exclude: [".vagrant/", ".git/", "__pycache__/"],
-      rsync_args: ["--verbose", "--ignore-from", "../.gitignore",
-                   "--archive", "--delete", "-z", "--copy-links"]
+      rsync__args: ["--verbose", "--exclude-from", "../.gitignore",
```
Comment (mgax) [inline ci/Vagrantfile]: You sure this works? I'd expect it would be affected by https://github.com/vmck/vagrant-vmck/issues/22

Comment (Jokeswar) [reply ci/Vagrantfile]: Yep, I checked it now, `./etc` and `./var` are empty just like `.gitignore` specified

Comment (mgax) [reply ci/Vagrantfile]: I don't like rsync using `.gitignore` for its ignore rules. They have different syntax, let's keep them separate. Please enumerate the excluded patterns here inline.

**Pull request merged**